### PR TITLE
Added in ability to have account limits configured in server config.

### PR DESCRIPTION
**Pull request by derekcollison**
Signed-off-by: Derek Collison <derek@nats.io>

/cc @nats-io/core

### CODE REVIEW

**Review by wallyqs — APPROVED** [review]
LGTM

File: server/accounts.go [hunk 1]
```diff
@@ -278,6 +278,8 @@ func (a *Account) shallowCopy() *Account {
 	}
 	// JetStream
 	na.jsLimits = a.jsLimits
+	// Server config account limits.
+	na.limits = a.limits
```
Comment (matthiashanel) [inline server/accounts.go:282]: Did we miss that for jwt before?

**Review by matthiashanel — APPROVED** [review]
Lgtm ,with a question to confirm my suspicion

File: server/accounts.go (hunk 1 above)
Comment (derekcollison) [reply server/accounts.go:282]: No JWT fills in when it process the claim, it pulls apart the limits from the JWTs and places them into the account.

https://github.com/nats-io/nats-server/blob/acc_config_limits/server/accounts.go#L3168

**Pull request merged**